contracts/deployables/strategies/StrategyV1.sol

```diff
@@ -29,59 +29,42 @@ contract StrategyV1 is
     address[] internal _votingAdapters;
     address[] internal _proposerAdapters;
 
+    modifier onlyProposalInitializer() {
+        if (msg.sender != _proposalInitializer)
+            revert InvalidProposalInitializer();
+        _;
+    }
+
     constructor() {
         _disableInitializers();
     }
 
     function initialize(
-        address azorius_,
+        address proposalInitializer_,
         uint32 votingPeriod_,
         uint256 quorumThreshold_,
         uint256 basisNumerator_,
         address[] memory votingAdapters_,
         address[] memory proposerAdapters_,
         address lightAccountFactory_
     ) public virtual override initializer {
-        if (azorius_ == address(0)) revert InvalidAzoriusAddress();
-
         __ERC4337VoterSupportV1_init(lightAccountFactory_);
-
-        _azorius = azorius_;
-
-        if (votingPeriod_ == 0) revert InvalidVotingPeriod();
+        _proposalInitializer = proposalInitializer_;
         _votingPeriod = votingPeriod_;
-
         _quorumThreshold = quorumThreshold_;
-
-        if (
-            basisNumerator_ >= BASIS_DENOMINATOR ||
-            basisNumerator_ < BASIS_DENOMINATOR / 2
-        ) revert InvalidBasisNumerator();
```

Comment:
I know we talked about removing initializer sanity checks - this is one I could see keeping since its enforcing a mathematical relationship

Reply:
great point, will put it back